backend/test/institution_request_collection_handler_test.py

```diff
@@ -76,3 +83,13 @@ def initModels(cls):
     cls.other_user.name = 'Other User'
     cls.other_user.email = ['otheruser@test.com']
     cls.other_user.put()
+    # new Institution Address
```

Comment:
We have a new approach to initialize mock objects. 

PR: https://github.com/eciis/web/pull/573